docs/ide4c/vscode/README.md

```diff
@@ -0,0 +1,7 @@
+# Configuring VS Code for ICU4C
+
+  - create `.vscode` folder in icu4c/source
```

Comment:
I prefer to work from one or even two folder levels higher up. For example, with config+make I always use out-of-source builds because I don't want to contaminate my source tree, and I often configure two or more variants (debug vs. release vs. hardcode-UTF-8...) in parallel into separate output folders.

Does this kind of thing make sense for VS Code?
Does it support out-of-source builds?

Reply:
We should aim at supporting one "suggested" workflow with these pre-built IDE files; people can customize them if they would like to do so.

I personally prefer my workspace root to be icu4c/source, largely because it reduces 2 levels of indentation in the file selector tree in the left panel of vscode.